pkg/new-kernel/kernel_config-5.15.x-aarch64

```diff
@@ -2685,9 +2688,9 @@ CONFIG_NET_VENDOR_SMSC=y
 CONFIG_NET_VENDOR_SOCIONEXT=y
 # CONFIG_SNI_NETSEC is not set
 CONFIG_NET_VENDOR_STMICRO=y
-CONFIG_STMMAC_ETH=m
+CONFIG_STMMAC_ETH=y
```

Comment:
stmmac is used in group with DWMAC_SUN8I and 0037 (yt8531c phy) patch for ethernet port. There are two options: 1) enable this module at boot,
2) Include it in the kernel.

I chose Include in the the kernel, because this is the main and only ethernet port on the board.

Reply:
I still don't see the needs of built-in here. You can build as a module and let it be loaded automatically by kernel during the booting. In this way we can avoid to include dwmac-sun8i driver in the kernel and loaded it in all ARM platforms (unnecessary).